web/index.html

```diff
@@ -58,5 +78,31 @@
 <body>
     <div style="position: absolute; top: 0; left: 0; right: 0; height: 30px;" id="drag-area"></div>
     <div id="root"></div>
+    <div id="splash" style="position: absolute; top: 0; left: 0; right: 0; background-color: white; width: 100%; height: 100%; display: flex; justify-content: center; align-items: center;">
+        <div class="splash-logo">
+            <svg fill="none" width="100%" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we not use `assets/images/new-expensify.svg` here instead (and ideally the logo for the correct environment)?

Reply:
I would rather just inline svg because of performance, again if we don't have webpack inlining this for us.